WeakAuras/WeakAuras.toc

```diff
@@ -29,6 +29,7 @@ Profiling.lua
 WeakAuras.lua
 Transmission.lua
 BuffTrigger.lua
+BuffTrigger2.lua
```

Comment:
You have translation phrases in this file, so babelfish needs to know about the file. Same with the Options file.